core/Makefile

```diff
@@ -88,6 +88,9 @@ test_rust: ## run rs unit tests
 test_emu: ## run selected device tests from python-trezor
 	$(EMU_TEST) $(PYTEST) $(TESTPATH)/device_tests $(TESTOPTS)
 
+test_emu_multicore: ## run device tests using multiple cores
+	$(PYTEST) -n auto $(TESTPATH)/device_tests $(TESTOPTS) --control-emulators --model=core --random-order-seed=$(shell echo $$RANDOM)
```

Comment:
is the explicit random seed so that all runners have the same one?
what happens when you leave it out?

Reply:
Yes, otherwise they would collect the tests in a different order and the master runner would complain about it.

Leaving it out produces:
```
pytest -n auto ../tests/device_tests/nem --control-emulators --model=core
============================================================================= test session starts ==============================================================================
platform linux -- Python 3.9.13, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
Using --random-order-bucket=module
Using --random-order-seed=20108

rootdir: /home/jmusil/trezor-firmware, configfile: setup.cfg
plugins: flaky-3.7.0, xdist-2.5.0, forked-1.4.0, ordering-0.6, random-order-1.0.4, asyncio-0.16.0, hypothesis-6.36.1, timeout-2.1.0
gw0 [17] / gw1 [17] / gw2 [17] / gw3 [17]
collecting 0 items / 1 error                                                                                                                                                   
==================================================================================== ERRORS ====================================================================================
_____________________________________________________________________________ ERROR collecting gw1 _____________________________________________________________________________
Different tests were collected between gw0 and gw1. The difference is:
--- gw0

+++ gw1

@@ -1,17 +1,17 @@

-tests/device_tests/nem/test_signtx_mosaics.py::test_nem_signtx_mosaic_supply_change
-tests/device_tests/nem/test_signtx_mosaics.py::test_nem_signtx_mosaic_creation_properties
....
```